lib/routes/account.js

```diff
@@ -52,6 +52,7 @@ module.exports = function (
         var authSalt = crypto.randomBytes(32)
         var authPW = Buffer(form.authPW, 'hex')
         var locale = request.app.acceptLanguage
+        var userAgent = request.headers['user-agent']
```

Comment:
Good point.

We could use something like [this](https://github.com/faisalman/ua-parser-js) to parse the UA string and then report it as `unknown` in the fallback case. Doing that would also allow us to store more readable data than vanilla UA strings.